screens/ResetPasswordScreen.js

```diff
@@ -0,0 +1,76 @@
+import React, { useContext, useEffect, useState } from "react";
+import { SafeAreaView, StyleSheet, View } from "react-native";
+import * as Yup from "yup";
+
+import AppButton from "../components/AppButton";
+import AppFormField from "../components/AppFormField";
+import runCrypto from "../auth/crypto-hashing";
+import defaultStyles from "../config/defaultStyles";
+import ErrorMessage from "../components/ErrorMessage";
+import Form from "../components/Form";
+import AuthContext from "../auth/Context";
+import SubmitButton from "../components/SubmitButton";
+
+const validationSchema = Yup.object().shape({
+  password: Yup.string().required().min(8).label("Password"),
+  confirmPassword: Yup.string().required().min(8).label("Confirm Password"),
+});
+
+export default function RegisterPassword() {
+  const [errorVisible, setErrorVisible] = useState(false);
+  const { user, setUser } = useContext(AuthContext);
+
+  useEffect(() => {
+    let unmounted = false;
+    return () => {
+      unmounted = true;
+    };
+  }, []);
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <Form
+        initialValues={{ password: "", confirmPassword: "" }}
+        onSubmit={async (values) => {
```

Comment:
Same for this function - it would be better above the return() statement, but still within the component.